src/script/components/calling/GroupVideoGrid.tsx

```diff
@@ -169,30 +169,32 @@ const GroupVideoGrid: React.FunctionComponent<GroupVideoGripProps> = ({
             }}
             srcObject={thumbnail.videoStream}
           />
-          <div className="group-video-grid__element__label" css={{padding: 4}}>
+          {/* <div className="group-video-grid__element__label" css={{padding: 4}}> */}
+          <div>
             {selfIsMuted ? (
               <span
-                className="group-video-grid__element__label__icon"
+                // className="group-video-grid__element__label__icon"
                 css={{'> svg': {width: 12}, height: 12}}
                 data-uie-name="status-call-audio-muted"
               >
-                <Icon.MicOff data-uie-name="mic-icon-off" />
+                {/* <Icon.MicOff data-uie-name="mic-icon-off" /> */}
               </span>
             ) : (
-              <ParticipantMicOnIcon
-                isActive={selfIsActivelySpeaking}
-                className="group-video-grid__element__label__icon"
-                css={{'> svg': {width: 12}}}
-              />
+              <span />
+              // <ParticipantMicOnIcon
+              //   isActive={selfIsActivelySpeaking}
+              //   className="group-video-grid__element__label__icon"
+              //   css={{'> svg': {width: 12}}}
+              // />
```

Comment:
Waiting for feedback on wether or not to have indication user is self muted on thumbnail